README.md

```diff
@@ -1,36 +1,53 @@
-# eZ Platform Developer Documentation
+# Ibexa DXP Developer Documentation
 
-This repository is the source for the [developer documentation for Ibexa DXP](https://doc.ibexa.co/en/latest),
+This repository is the source for the [developer documentation for Ibexa](https://doc.ibexa.co/en/latest),
 a DXP platform that is based on the Symfony Full Stack Framework in PHP.
 
 # Resources
 
 1. Ibexa DXP Developer Hub: https://developers.ibexa.co
-1. Ibexa DXP Repository: https://github.com/ezsystems/ezplatform
+1. Ibexa DXP Repository: https://github.com/ibexa/oss
 1. Open JIRA board: https://issues.ibexa.co
 1. Ibexa Website: https://ibexa.co
 1. User documentation: https://doc.ibexa.co/projects/userguide
 
 ## How to contribute
-https://doc.ibexa.co/en/master/community_resources/contributing
 
-Invite yourself to the community Slack and join `#documentation-contrib`
+If you want to contribute to the documentation, you can open PR in this repository.
 
-https://ez-community-on-slack.herokuapp.com/
+If you'd like to see Ibexa DXP in your language, you can [contribute to the translations](https://doc.ibexa.co/en/master/resources/contributing/contribute_translations/).
 
-## Building & preview
-To build and preview your changes locally, you will need Python along with its package manager (`pip`).
+### Contribute to API reference
+
+REST API Reference is located in the `docs/api/rest_api/rest_api_reference/rest_api_reference.html`
+and is generated automatically by the RAML2HTML tool.
+It is based on `*.raml` files located in `docs/api/rest_api/rest_api_reference/input` directory that you can edit in your editor/IDE.
+
+After you modify relevant files in the input folder, you can generate an HTML file from repository root (this step can also be performed by one of the Tech Writers during PR review): 
+
+`php tools/raml2html/raml2html.php build --non-standard-http-methods=COPY,MOVE,PUBLISH,SWAP -t default -o docs/api/rest_api/rest_api_reference/output/ docs/api/rest_api/rest_api_reference/input/ez.raml`
+
+In case of errors, look for mistakes in RAML file, for example, double apostrophes.
+Move `rest_api_reference.html`  from the output folder to `docs/api/rest_api/rest_api_reference/` root.
+
```

Comment:
```suggestion
Move `rest_api_reference.html`  from the output folder to `docs/api/rest_api/rest_api_reference/` root.

See tools/raml2html/README.md for more.

```

Reply:
But PR mentioned by you is still open, we can add a link to it once it's merged